prusti-specs/src/extern_spec_rewriter.rs

```diff
@@ -226,18 +228,51 @@ pub fn generate_new_struct(item: &syn::ItemImpl) -> syn::Result<syn::ItemStruct>
     new_struct.generics = item.generics.clone();
     let generics = &new_struct.generics;
 
-    let mut fields_str: String = String::new();
+    let mut fields = CommaPunct::<syn::Field>::new();
 
     // Add `PhantomData` markers for each type parameter to silence errors
     // about unused type parameters.
     for param in generics.params.iter() {
-        let field = format!("core::marker::PhantomData<{}>,", param.to_token_stream().to_string());
-        fields_str.push_str(&field);
+        let field: ParsableUnnamedField = match param {
+            syn::GenericParam::Type(tp) => {
+                let ident = tp.ident.clone();
+                syn::parse_quote!{ core::marker::PhantomData<#ident> }
```

Comment:
Minor thing to prevent user-defined `core` imports or something…
```suggestion
                syn::parse_quote!{ ::core::marker::PhantomData<#ident> }
```
Also in L243.